plugins/update-po-files.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function (grunt) {
+
+    'use strict';
+
+    const execSync = require('child_process').execSync;
+
+    const task = function () {
+        const msgmergeLookup = 'which msgmerge;echo $?';
+        const msgmergePresent = !parseInt(execSync(msgmergeLookup, 'utf8'), 10);
+
+        if (!msgmergePresent) {
+            grunt.fatal(
+                'msgmerge is not available on the system. Please make sure gettext is' +
+                ' installed. You may have to do \'brew link gettext --force\' if on OS X.'
+            );
+        }
+
+        this.options({
+            template: 'template.pot'
+        });
+
+        const templateFile = this.options().template;
+
+        if (!this.filesSrc.length) {
+            grunt.fatal('No *.po files found');
```

Comment:
I don't think you need to fatally quit if no files are found. You can just log how many files are worked on, even in cases where 0 files are worked on. This plugin should be able to be added without any po files, otherwise devs would have to keep it in their mind to add it only when they get po files, and then remove it when they don't have any.